dev/ToastNotifications/ToastProgressDataABI.cpp

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+#include "ToastProgressData.h"
+#include "ToastProgressDataABI.h"
+#include "ToastNotificationUtility.h"
+
+namespace winrt
+{
+    using namespace winrt::Microsoft::Windows::ToastNotifications;
+}
+
+namespace ToastABI
+{
+    using namespace ::ABI::Microsoft::Internal::ToastNotifications;
+}
+
+ToastProgressDataABI::ToastProgressDataABI(winrt::ToastProgressData const& progressData)
+{
+    m_progressData = progressData;
+}
+
+STDMETHODIMP_(HRESULT __stdcall) ToastProgressDataABI::get_SequenceNumber(_Out_ unsigned int* value)
```

Comment:
noexcept for all methods returning hresult

Reply:
Isn't that implied by being WinRT methods?

I thought coding guidelines was to minimize use of noexcept, mainly boiling down to use noexcept for Flat-C APIs but not WinRT or implementation details.

@jonwis what's the latest guidance?

Reply:
@hulumane @jonwis what is the latest guidance here?